apps/ledger-live-mobile/src/screens/PairDevices/index.tsx

```diff
@@ -203,7 +206,9 @@ function PairDevicesInner({ navigation, route }: NavigationProps) {
         } finally {
           transport.close();
```

Comment:
[NOTE]: This part here should probably be disabled in the context of ldmk as we don't need to close or disconnect the device.